Add migration section regarding URL configuration (#374)
CloudCannon/platform-documentation: guides/unified-configuration-migration-guide/build-configuration-and-conclusion.mdx

<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/builds/configure-your-first-build/2024-08/CloudCannon-Documentation-Site-Settings-Build-Configuration-Setup.png" alt="A screenshot of the Build Configuration page shows blue boxes containing CloudCannon's suggestions for your command line options." title="The Build Configuration page" type="screenshot"/>

Finally, build your Site to remove any existing build integration output from previous builds. If you updated your build commands, CloudCannon will automatically trigger a build for you.

<comp.Notice info_type="info">
  Even if you are planning to switch to [Headless Mode](/documentation/articles/what-is-headless-mode/), we recommend completing a final build before you switch.
</comp.Notice>

## URL configuration

> For Eleventy, Hugo, and Jekyll Sites, CloudCannon discovers URLs through a different mechanism when building with Unified Configuration.
> 
> Previously, the output URL for a file was read from your SSG at build time. Instead, CloudCannon now determines output URLs using heuristics after building your Site.
> 
> Most Sites will be unchanged, however some Sites may fail to auto-detect the correct URLs. We recommend checking that the URLs displayed in CloudCannon are correct.
> 
> URLs are used when copying the output path of a collection item, and when opening the Visual Editor for a file. If you experience any issues with these actions, we recommend [configuring explicit URLs for your collections](/documentation/articles/define-your-collections/#url).

Reviewer: ```suggestion
For Eleventy, Hugo, and Jekyll Sites, CloudCannon discovers URLs through a different mechanism when building with Unified Configuration.

Previously, CloudCannon would read output URLs from your SSG at build time. Using Unified Configuration, CloudCannon now determines output URLs using heuristics after building your Site.

This won't affect most Sites, however CloudCannon may fail to auto-detect the correct URLs. We recommend checking that the URLs displayed in CloudCannon are correct.

CloudCannon also uses URLs when copying the output path of a collection item and when opening the Visual Editor for a file. If you experience any issues with these actions, we recommend [configuring explicit URLs for your collections](/documentation/articles/define-your-collections/#url).
```
